docs/source/documentation/user_guide.rst

```diff
@@ -9,4 +9,5 @@ User guide
     debug.rst
     gpu.rst
     substra_tools.rst
-    get_performances.rst
\ No newline at end of file
+    get_performances.rst
```

Comment:
I think we are again reaching the point of reorganizing this documentation - These pages don't seem to target the same audience

Reply:
Agreed